test-series/0/Validation_040.ipynb

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
<p>Maybe I'm looking too hard, but it seems that in each of the panels, there are short sections of realization number where all four standard deviations have estimates that are above/below the truth (eg. at about N=110 in the left-hand plot). Of course, all things statistical are possible, but is this considered highly unlikely (it looks like it does this for ~5-6 realizations in a row). Are you sure there are no correlations between runs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>